apps/radioss/run_T10M.pbs

```diff
@@ -0,0 +1,125 @@
+
+## RADIOSS Options
+# Define HW_ROOT (location HyperWorks is installed)
+HW_BASE=$INSTALL_DIR
+HW_ROOT=${HW_BASE}/radioss/bin/linux64
+
+starter=$HW_ROOT/s_2018.0.1_linux64
+engine=$HW_ROOT/e_2018.0.1_linux64_impi
+user=$USER
+
+## Data Options
+ROOTNAME=TAURUS_A05_FFB50
+d00=${ROOTNAME}_0000.rad
+d01=${ROOTNAME}_0001.rad
+MPIBIN=/opt/intel/impi/2018.4.274/intel64/bin
+MPILIB=/opt/intel/impi/2018.4.274/intel64/lib
+
+# JOB Options
+export ALTAIR_LICENSE_PATH=6200@$LIC_SRV
+cp ${HW_BASE}/common/bin/linux64/radflex_2018_linux64 ./radflex_2018_linux64
+export RADFLEX_PATH=${HW_BASE}/common/bin/linux64/
+export LD_LIBRARY_PATH=${HW_ROOT}/:$RADFLEX_PATH:$LD_LIBRARY_PATH
+
+wkdir=$DATA_DIR/projects/${USER}/radioss-$PBS_JOBID
+mkdir -p $wkdir
+cd $wkdir
+ln -s $DATA_DIR/radioss/T10M/* .
+unlink includes
+cp -a $DATA_DIR/radioss/T10M/includes .
+echo Local execution under: `pwd`
+
+# run RADIOSS starter
+echo "------------------------------------------------------"
+echo "RADIOSS Starter"
+echo "------------------------------------------------------"
+ulimit -s unlimited
+$starter -np $NMPI -i $d00
+# run RADIOSS engine
+# this part is needed if using local ssd for the wkdir
+if [[ $wkdir == *"/mnt/resource/"* ]]; then
+    n=1
+    for i in `cat $PBS_NODEFILE`
+    do
+        ssh $i test -f $wkdir/$d01
+        a=$?
+        if [ $a -ne 0 ]; then
+            ssh $i mkdir $wkdir
+            scp $d01 $i:$wkdir/$d01
+        fi
+        if [ $n -lt 10 ]; then
+            scp ${ROOTNAME}_0000_000$n.rst $i:$wkdir/${ROOTNAME}_0000_000$n.rst;
+        elif [ $n -lt 100 ]; then
+            scp ${ROOTNAME}_0000_00$n.rst $i:$wkdir/${ROOTNAME}_0000_00$n.rst;
+        elif [ $n -lt 1000 ]; then
+            scp ${ROOTNAME}_0000_0$n.rst $i:$wkdir/${ROOTNAME}_0000_0$n.rst;
+        else
+            scp ${ROOTNAME}_0000_$n.rst $i:$wkdir/${ROOTNAME}_0000_$n.rst;
+        fi
+        n=$(($n+1))
+    done
+fi
+
+echo "------------------------------------------------------"
+echo "RADIOSS Engine"
+echo "------------------------------------------------------"
+
+
+echo "$MPIBIN/mpirun -r ssh -machinefile $PBS_NODEFILE -genv I_MPI_MPIRUN_CLEANUP 1 -genv I_MPI_PIN_DOMAIN auto -genv OMP_NUM_THREADS $OMP_NUM_THREADS -genv I_MPI_ADJUST_BCAST 1 -genv I_MPI_ADJUST_REDUCE 2 -genv KMP_AFFINITY verbose,scatter -genv KMP_STACKSIZE 400m -genv I_MPI_DEBUG 3 -genv I_MPI_FABRICS shm:ofa -genv LD_LIBRARY_PATH $MPILIB:$LD_LIBRARY_PATH -genv PATH $MPIBIN:$PATH -n $NMPI /opt/pbs/bin/pbs_attach -j $PBS_JOBID -s $engine  -i $d01"
+$MPIBIN/mpirun -r ssh -machinefile $PBS_NODEFILE -genv I_MPI_MPIRUN_CLEANUP 1 -genv I_MPI_PIN_DOMAIN auto -genv OMP_NUM_THREADS $OMP_NUM_THREADS -genv I_MPI_ADJUST_BCAST 1 -genv I_MPI_ADJUST_REDUCE 2 -genv KMP_AFFINITY verbose,scatter -genv KMP_STACKSIZE 400m -genv I_MPI_DEBUG 3 -genv I_MPI_FABRICS shm:ofa -genv LD_LIBRARY_PATH $MPILIB:$LD_LIBRARY_PATH -genv PATH $MPIBIN:$PATH -n $NMPI /opt/pbs/bin/pbs_attach -j $PBS_JOBID -s $engine  -i $d01
```

Comment:
Can you document where you got the flags from?  Also, is there any backup for them running faster than the defaults?  FYI we often find dapl performs better than ofa - have you tested this?

Reply:
This run script came from the vendor. Do you have any better suggestions on which flags to use? I can give them a try and we can use which ever ones perform better.